app/controllers/regular_events/calendars_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class RegularEvents::CalendarsController < ApplicationController
+  skip_before_action :require_active_user_login, raise: false, only: :index
+
+  def index
+    user_id = params[:user_id]
+    user = User.find_by(id: user_id)
+    respond_to do |format|
+      format.ics do
+        regular_calendar = RegularEventsToIcalExporter.export_events(fetch_events(user))
+        render plain: regular_calendar.to_ical
+      end
+    end
```

Comment:
扱っているフォーマットが単一なので、`respond_to`を使うメリットが少ないよう感じました。

```suggestion
    regular_calendar = RegularEventsToIcalExporter.export_events(fetch_events(user))
    render plain: regular_calendar.to_ical
```